### PR TITLE
Make background image scale to fit page

### PR DESCRIPTION
**Pull request by Wrenderer**
* Remove background from `#main` and move it to `#wrap` as increasing the height of the page (i.e. on an infinite feed page) will move the background if on `#main`
 * Add background to main `html` tag so that the fixed background attachment will work on mobile/small screens
 * Modify the function that changes backgrounds to change both `html` and `#wrap` instead of `#main`.
    * Two backgrounds are necessary because the hamburger mobile menu is hidden underneath `#wrap` but on top of `html`

[Finished: #95492834]

### CODE REVIEW

File: app/assets/stylesheets/sunnyside.css.scss [hunk 1]
```diff
@@ -7,9 +7,10 @@ a.navbar-brand {
   padding: 0px;
 }
 
-#main {
-  background-size: 100%;
+html, #wrap {
+  background-size: cover;
```
Comment (eschlange) [inline app/assets/stylesheets/sunnyside.css.scss]: I have never used this before. Looks very useful for our circumstances.

File: app/views/_background_select.html.erb [hunk 2]
```diff
@@ -38,7 +38,9 @@
       url: '/customize_view/update_background',
       data: ({"participant_id":"<%= current_participant ? current_participant.id : "" %>","background_image":background_image}),
       success: function(response, status) {
-        $("#main").css("background-image", "url("+background_path+")");
+        $("html").css("background-image", "url("+background_path+")");
+        console.log(status);
```
Comment (eschlange) [inline app/views/_background_select.html.erb]: Let's remove console logging before pushing to master.

File: app/views/_background_select.html.erb [hunk 3]
```diff
@@ -38,7 +38,9 @@
...
+        $("#wrap").css("background-image", "url("+background_path+")");
         console.log(status);
```
Comment (eschlange) [inline app/views/_background_select.html.erb]: Please remove~

Comment (Wrenderer) [inline app/views/_background_select.html.erb]: Ah, good point

**Comment by Wrenderer** [comment]
`console.log` removed

**Comment by eschlange** [comment]
:+1:

**Pull request merged**